src/docs/sdk/rate-limiting.mdx

```diff
@@ -83,6 +92,7 @@ While these [data categories](https://github.com/getsentry/relay/blob/master/rel
   - `session`: Session update events
   - `profile`: Profiling events
   - `replay`: Session Replays
+  - `metric_bucket`: Sentry Metrics sent via the `statsd` or `metrics` items. The _namespace_ component of the quota limit defines which namespace is affected.
```

Comment:
```suggestion
  - `metric_bucket`: Sentry Metrics sent via the `statsd` or `metrics` items. The `namespace` component of the *quota_limit* defines which namespace is affected.
```

Just to make it a little bit more clear to what we are referring to here since we call it "quota_limit" above and namespace is a qualifier.